AnnoyancesFilter/sections/popups.txt

```diff
@@ -117,6 +117,9 @@ sodapdf.com##.be-tooltip
 ! Specific sites
 !######### Subscription popups add to subscriptions.txt #########
 !
+biz.moneyforward.com##.advads-background
+biz.moneyforward.com##.money-popup-sp 
+biz.moneyforward.com##.money-popup-pc
```

Comment:
The last two rules of money-popup- can be merged into `biz.moneyforward.com##.advads-effect.is-sticky`

Reply:
fixed